### ISSUE TITLE
components/Chart/events - pickEventName will raise `invalid group specifier name` on Safari

### ISSUE THREAD

**Issue by msyfls123** [alibaba/BizCharts#1182]
On Safari and mobile Chrome.
BizCharts: 4.0.5
React: 16.13.1

https://github.com/alibaba/BizCharts/blob/268c33cf32e52609db30113242a8651badc924b2/src/components/Chart/events.ts#L38
```bash
> /(?<=on).*/.exec('dasd')
> SyntaxError: Invalid regular expression: invalid group specifier name
```
Also, since I have tried to use Interval separately, I will encounter such an error ...

![image](https://user-images.githubusercontent.com/7849435/84226787-5233a280-ab15-11ea-9eca-a1cd315d3170.png)
https://github.com/alibaba/BizCharts/blob/268c33cf32e52609db30113242a8651badc924b2/src/geometry/Base.tsx#L30
`Uncaught TypeError: Cannot read property 'rootChart' of null`

**Comment by tli4** [comment]
plz fix this 🙏

**Comment by Leannechn** [comment]
fix in 4.0.6
